CourseDevelopment/Week1/CODE_OF_CONDUCT_template.md

```diff
@@ -0,0 +1,60 @@
+ 
+#### Characteristics we value:  
+
+
+#### Behaviors we encourage
+
+
+#### Behaviors we discourage? 
+
+	
+#### We make each others feel safe and supported by
+
+    
+#### How to report an issue, should someone violate the code? 
+
+1. Contact Sanli or Erik by private message, or in person. All communication will be treated as confidential. 
+2. If for any reason you don’t want to do 1, you can contact *THIRD PERSON to be assigned* who has agreed to function as external contact person for this purpose.
+
+---
+
+## Community Participation Guidelines
+Apart from the code of conduct, all participants in this course must follow these general guidelines.
+
+### Participation Guidelines
+
+When participating in OS4P course, respect the Utrecht University guideline for [academic and scientific integrity](https://www.uu.nl/en/files/netherlands-code-of-conduct-for-research-integrity2018ukpdf) and [code of conduct](https://www.uu.nl/en/files/codeofconductuuenpdf). These guidelines cover our behaviour as participants, mentors, experts, staff, volunteers, and anyone else involved in making this course possible.
+
+### How to treat each other
+
+To create a collaborative and inviting learning environment, we also emphasise certain values in how we treat each other:
+
++ Be respectful and value each other’s ideas, styles and viewpoints.  
++ Be direct but professional; we cannot withhold hard truths.  
++ Be inclusive and help new perspectives be heard.  
++ Appreciate and accommodate our many cultural practices, attitudes and beliefs.  
++ Be open to learning from others.  
++ Lead by example and match your actions with your words.  
+
+The following will not be tolerated during the activities related to this course: 
+
++ violence and threats of violence
++ personal attacks; derogatory language
++ unwelcome sexual attention or physical contact
++ disruptive behaviour
++ influencing unacceptable behaviour.
+
+### Inclusion and Diversity
+
+We welcome contributions from everyone as long as they interact constructively with our community, including, but not limited to people of varied age, culture, ethnicity, gender, gender-identity, language, race, sexual orientation, geographical location and religious views.
+
+### Raising Issues
+If you believe you‘re experiencing practices which don‘t meet the above policies, or if you feel you are being harassed in any way, please immediately contact the course coordinator Sanli Faez, or the university Academic Integrity Counsellor. [Prof. Liesbeth Woertman](https://www.uu.nl/staff/emwoertman&t=0).
+
+The course coordinator reserves the right to refuse admission to anyone violating these policies, and/or take further action including reporting to the responsible figure for academic integrity at the department. 
```

Comment:
Also include the exam committee here? Could there be repercussions like not passing the course?